apps/site/public/types/modules/graph/0.3/schema/subgraph.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://blockprotocol.org/types/modules/graph/0.3/schema/subgraph",
+  "title": "Subgraph",
```

Comment:
We should add a `$comment` saying this is a non-exhaustive type and users should consult the documentation and/or the `blockprotocol/graph` package

Reply:
[Add comment on non-exhaustiveness for the subgraph schema](https://github.com/blockprotocol/blockprotocol/pull/1034/commits/a9db89eec4890e960c64291327a4de5a2662e471)